inst/tutorials/data_visualization_basic/data_visualization_basic.Rmd

```diff
@@ -0,0 +1,267 @@
+```{r dot1, exercise=TRUE}
+ggplot(dat, aes(x = Depth, y = CTD_O2))
+```
+
+*NOTE: The x-axis showing depth is not linear because we converted depth to factors.*
+
+Finally we add the geom to specify how we want to map our data onto these axes (*i.e.* points, boxplot, lines, etc.). Here, we will use points for the data. 
+
+Plot depth and oxygen:
+```{r depthplt, exercise=TRUE}
+ggplot(dat, aes(x = Depth, y = CTD_O2)) +
+  geom_point()
+```
+
+Importantly, aesthetics in the `ggplot` layer will be applied to all layers while those in a specific geom will only be applied to that one layer. So in the above code, we have given aesthetics in `ggplot` so they will be used in `ggplot` and `geom_point` (and any other additional layers we might add). 
+
+In contrast, in the code below, we specify the aesthetics in the geom so only `geom_point` will use it. In this case, the plots are the same.
+```{r depthplta, exercise=TRUE}
+ggplot(dat) +
+  geom_point(aes(x = Depth, y = CTD_O2))
+```
+
+Let's check your understanding by visualizing the realtionship between depth and methane using a dot plot. The resulting plot should look like this:
+
+```{r plotexample, echo = FALSE, warning = FALSE}
+ggplot(dat, aes(x = Depth, y=Mean_CH4)) +
+  geom_point()
+```
+
+```{r exercise1, exercise=TRUE}
+ggplot(dat, aes(x = , y=))
+```
+
+Since you only specified the minimum pieces, the rest are filled in with `ggplot2` defaults.
+
+Now, you can add pieces to alter these defaults. Below, see how we keep adding to the plot with additional pieces to:
+
+Change axes labels
+```{r labels, exercise=TRUE}
+ggplot(dat, aes(x= Depth, y = CTD_O2)) +
+  geom_point() +
+  labs(x="Depth [m]", y="Oxygen [uM]")
+```
+
+Change the point color. You can specify a specific color like "orange":
+```{r example, exercise=TRUE}
+ggplot(dat, aes(x= Depth, y = CTD_O2)) +
+  geom_point(color = "orange") +
+  labs(x="Depth [m]", y="Oxygen [uM]")
+```
+
+Build onto the previous plot visualizing depth and methane by: 
+1. changing the axis labels to Depth [m] and Methane [uM]
+2. specifying the points to be blue
+
+```{r plotexample2, echo = FALSE, warning = FALSE}
+ggplot(dat, aes(x = Depth, y=Mean_CH4)) +
+  geom_point(color="blue") +
+  labs(x="Depth [m]", y="Methane [uM]")
+```
+
+```{r exercise2, exercise=TRUE}
+ggplot(dat, aes(x = , y=))
+```
+
+There are 7 data points for each depths, but a few of them fall in the same range and cannot be resolved visulally (the data was *overplotted*). One way to deal with overplotting is to use transparency, denoted by the `alpha` argument.
+
+```{r example3, exercise=TRUE}
+ggplot(dat, aes(x = Depth, y = CTD_O2)) +
+  geom_point(alpha = 0.5) +
+  labs(x="Depth [m]", y="Oxygen [uM]")
+```
+
+Change the point shape. Similar to color, this can be a single shape or mapped to a variable:
```

Comment:
Mapping colour to a variable wasn't addressed above, so might be best to not bring it up here. You could also cover it in the previous section, though, since it is useful to know.